core/src/components/progress-bar/progress-bar.ionic.scss

```
@@ -0,0 +1,22 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./progress-bar";
+
+// Ionic Progress bar
+// --------------------------------------------------
+
+:host {
+  --background: #{globals.$ion-bg-neutral-subtle-default};
```

Why was the value changed? The designs show neutral 100.

```suggestion
  --background: #{globals.$ion-primitives-neutral-100};
```

---

Sorry. I updated the link in the task but could have been a bit clearer here. Please check the design link and you should see that design token being used.